tests/visibility_negative/visibility_negative.fz.expected_err

```diff
@@ -2312,69 +2312,69 @@ Target feature: 'visibility_negative.visi1.a2.b3.c6'
-Actual type for argument #1 'b' does not match expected type.
-In call to          : 'visibility_negative.chck'
-expected formal type: 'bool'
-actual type found   : '**error**'
-assignable to       : '**error**'
-for value assigned  : ''
-To solve this, you could change the type of 'b' to a 'ref' type like 'ref bool'.
-
-
---CURDIR--/visibility_negative.fz:526:5: error 291: Could not find called feature
+--CURDIR--/visibility_negative.fz:526:5: error 290: Could not find called feature
     b // 273. should flag an error: feature not visible
 ----^
 Feature not found: 'b' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'b'
 
 
---CURDIR--/visibility_negative.fz:527:5: error 292: Could not find called feature
+--CURDIR--/visibility_negative.fz:527:5: error 291: Could not find called feature
     c // 274. should flag an error: feature not visible
 ----^
 Feature not found: 'c' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'c'
 
 
---CURDIR--/visibility_negative.fz:530:9: error 293: Could not find called feature
+--CURDIR--/visibility_negative.fz:530:9: error 292: Could not find called feature
         c // 275. should flag an error: feature not visible
 --------^
 Feature not found: 'c' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'c'
 
 
---CURDIR--/visibility_negative.fz:533:9: error 294: Could not find called feature
+--CURDIR--/visibility_negative.fz:533:9: error 293: Could not find called feature
         c // 276. should flag an error: feature not visible
 --------^
 Feature not found: 'c' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'c'
 
 
---CURDIR--/visibility_negative.fz:536:9: error 295: Could not find called feature
+--CURDIR--/visibility_negative.fz:536:9: error 294: Could not find called feature
         b // 277. should flag an error: feature not visible
 --------^
 Feature not found: 'b' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'b'
 
 
---CURDIR--/visibility_negative.fz:539:9: error 296: Could not find called feature
+--CURDIR--/visibility_negative.fz:539:9: error 295: Could not find called feature
         b // 278. should flag an error: feature not visible
 --------^
 Feature not found: 'b' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'b'
 
 
---CURDIR--/visibility_negative.fz:541:5: error 297: Could not find called feature
+--CURDIR--/visibility_negative.fz:541:5: error 296: Could not find called feature
     b // 279. should flag an error: feature not visible
 ----^
 Feature not found: 'b' (no arguments)
 Target feature: 'visibility_negative.visi8'
 In call: 'b'
 
+
+<built-in>: error 297: Incompatible types when passing argument in a call
+Actual type for argument #1 'b' does not match expected type.
+In call to          : 'visibility_negative.chck'
+expected formal type: 'bool'
+actual type found   : '**error**'
```

Comment:
Why is this error here now?

Reply:
What I remember this could not be avoided, I think we even talked about it. But I don't remember what it was and I could not find the code that causes it again now